src/_includes/game.html

```diff
@@ -21,11 +21,14 @@
     {%- continue -%}
 {%- endif -%}
 
-<li><h4>{{ include.game }}
+{% unless include.notitle %}
```

Comment:
we should put dashes on the % signs, to prevent rendering a bunch of extra whitespace (like when you view page source)

it's not a big deal but I guess a lot of extra new lines and spaces could bloat the download size slightly

```suggestion
{%- unless include.notitle -%}
```

Reply:
converted to draft until this is ready for review again